src/App.vue

```diff
@@ -432,6 +431,10 @@ export default {
 		 * first navigation will be made from initial state { name : undefined }
 		 */
 		this.$router.beforeEach((to, from, next) => {
+			if (to.fullPath === from.fullPath) {
+				// Block duplicated navigation
+				return
+			}
```

Comment:
```suggestion
			if (isNavigationFailure(failure)) {
				// Block failed navigation
				return
			}
```

https://router.vuejs.org/api/functions/isNavigationFailure.html